apis/externalsecrets/v1beta1/secretstore_beyondtrust_types.go

```diff
@@ -0,0 +1,50 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
+
+type BeyondTrustProviderSecretRef struct {
+
+	// Value can be specified directly to set a value without using a secret.
+	// +optional
+	Value string `json:"value,omitempty"`
```

Comment:
If we are allowing for sensitive data to be exposed directly via manifests (such as `ClientSecret` or the `CertificateKey`, I'd say this should be a harder name for users, such as `insecureValue`.

Either that or separate for sensitive information to not use this type, and instead use just `SecretRef`.

Reply:
Yes, we are creating kubernetes secrets for those sensitive data, like next:

`kubectl create secret generic bt-secret --from-literal ClientSecret=*********
`
We are not exposing sensitive data directly via manifests.